standards/learning-resources.md

```diff
@@ -0,0 +1,66 @@
+# <a name="resources"></a>Learning Resources
+
+If you are interested in learning Ruby then that's fantastic!  Below you'll find a curated list of some of the best resources for getting started:
+
+1. [Nerdery Getting Started Guide](https://ruby.nerderylabs.com/training/training-module-0/) - for getting started.
+1. [Code School](https://www.codeschool.com/learn/ruby) - fun, interactive video lessons.
+1. [Ruby Warrior](https://www.bloc.io/ruby-warrior/) - a game for learning Ruby.
+1. [Rails Best Practices](http://rails-bestpractices.com/) - for learning a *right* way.
+1. [Rails Guides](http://guides.rubyonrails.org/) - the ultimate API document for figuring Rails out.
+1. [Ruby Koans](http://rubykoans.com/) - learn the quirks of Ruby by fixing unit tests.
+
+# Installing Ruby on Rails
+
+Credit: Jordan's [Rails Best Practices](http://www.phantomdata.com/2016/07/21/rails-best-practices.html) guide.
+
+We work in a company with lots of different disciplines all commingling on a regular basis.  PHP, .NET, Java, Rails and Python...  we all get along pretty well.  We're also quite adaptable and end up going where the work is - which might not necessarily be in our favorite discipline.  The most common frustrations I see developers thrown into the Rails world is that of not having started from a good base.
+
+You see, Ruby is pretty popular.  You might not know it, but a metric ton of the critical tools you use daily are built upon it.  For the most part, unless you're running Windows, Ruby is already installed.  ***gasp*** it was that easy?
```

Comment:
Can likely add sources to back this.  I like to point out that Ruby was recently reintroduced to the top 10 programming languages, and even the top 20 actually means it has somewhat significant usage in the world.